nessus-agent/windows/init.sls

```diff
@@ -6,45 +6,21 @@
 # In addiont, the sls file requires that the salt winrepo has been
 # configured with a `nessus agent` pkg definition
 
-{%- if nessus.log_rotation_strategy %}
-Add Log Rotation Strategy:
+{%- for log_file,log_rotation in nessus.log_config.items() %}
+  {%- if log_rotation %}
+    {%- for rotation_param,rotation_value in log_rotation.items() %}
+      {%- if rotation_value %}
+Add Log Config {{ log_file }} {{ rotation_param }} {{ rotation_value }}:
```

Comment:
You don't need the rotation_value in the state name...

Reply:
Agree.  I'll remove it.